From c77b09fe37ac00c86e6ac43db5e43b325fa4d8a1 Mon Sep 17 00:00:00 2001
From: Carl Schwan <schwancarl@protonmail.com>
Date: Thu, 14 Mar 2019 23:24:27 +0100
Subject: Follow KDE HIG by using ellipis

Summary:
See https://hig.kde.org/style/writing/placeholder.html#actions

@stikonas I add you as reviewer because I saw that you are last contributor to KTorrent :D

Test Plan: Build and run

Reviewers: #vdg, stikonas, ngraham

Reviewed By: #vdg, stikonas, ngraham

Subscribers: stikonas

Maniphest Tasks: T10258

Differential Revision: https://phabricator.kde.org/D19772
---
 ktorrent/view/torrentsearchbar.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/ktorrent/view/torrentsearchbar.cpp b/ktorrent/view/torrentsearchbar.cpp
index 045d695..2e9c428 100644
--- a/ktorrent/view/torrentsearchbar.cpp
+++ b/ktorrent/view/torrentsearchbar.cpp
@@ -46,7 +46,7 @@ namespace kt
 
         search_bar = new QLineEdit(this);
         search_bar->setClearButtonEnabled(true);
-        search_bar->setPlaceholderText(i18n("Torrent filter"));
+        search_bar->setPlaceholderText(i18n("Filter..."));
         connect(search_bar, SIGNAL(textChanged(QString)), view, SLOT(setFilterString(QString)));
         connect(this, SIGNAL(filterBarShown(QString)), view, SLOT(setFilterString(QString)));
 
-- 
cgit v1.1
